.yamllint.yaml

```diff
@@ -0,0 +1,8 @@
+extends: default
+
+rules:
+  document-start: disable
+  line-length:
+    max: 120
```

Comment:
This is because at the default (80) there were a large number of issues. At this value there was only one and it was feasibly addressable.

Reply:
(No longer valid because I'm using the starbase yamllint config)